src/storage/patch/RepresentationPatchHandler.ts

```diff
@@ -41,6 +43,13 @@ export class RepresentationPatchHandler extends PatchHandler {
     // Patch it
     const patched = await this.patcher.handleSafe({ patch, identifier, representation });
 
+    // Solid, §5.3: "Servers MUST NOT allow HTTP POST, PUT and PATCH to update a container’s resource metadata
+    // statements; if the server receives such a request, it MUST respond with a 409 status code.
+    // https://solid.github.io/specification/protocol#contained-resource-metadata-statements
+    if (isContainerIdentifier(identifier)) {
+      throw new ConflictHttpError('Not allowed to execute PATCH request on containers.');
+    }
```

Comment:
I think we still have a miscommunication here. The comment above this function is saying that certain specific metadata triples can not be modified (specifically the triples that need to be checked in the new patcher being added in this PR). It does not say that you can't modify container metadata.

The reason you can't PATCH a container is simply because we internally decided to only allow modifying a container's metadata through its metadata resource, not through the resource itself to have some consistency.

Additional fun fact: it is currently impossible for this statement to ever trigger at all in practice, since if you look at the `patching.json` config, you'll see there is a `ContainerPatcher` being defined there that will catch all requests targeting a container. That part of the config will have to be removed.